policyengine_us/variables/gov/states/co/tax/income/credits/income_qualified_senior_housing/co_income_qualified_senior_housing.py

```diff
@@ -0,0 +1,44 @@
+from policyengine_us.model_api import *
+
+
+class co_income_qualified_senior_housing(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Colorado Income Qualified Senior Housing Income Tax Credit"
+    unit = USD
+    definition_period = YEAR
+    reference = (
+        "https://tax.colorado.gov/income-qualified-senior-housing-income-tax-credit",
+        "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=17",
+    )
+    defined_for = "co_income_qualified_senior_housing_eligible"
+
+    def formula(tax_unit, period, parameters):
+        p = parameters(
+            period
+        ).gov.states.co.tax.income.credits.income_qualified_senior_housing
+        filing_status = tax_unit("filing_status", period)
+        status = filing_status.possible_values
+        agi = tax_unit("adjusted_gross_income", period)
+
+        # in the instruction, it didn't provide any information about widown
+        # "https://tax.colorado.gov/sites/tax/files/documents/DR_104_Book_2022.pdf#page=17"
+        # I consider widow is the same as the single, joint and hoh
```

Comment:
is there in the tax form a mention about widow in general or sometimes on the website of the credit they have a FAQ

Reply:
Instructions and tax form doesn't mentioned anything about widow. I feel like widow should be the same as hoh in Colorado since the only thing they mentioned about widow was "Retirement Pension or Annuity Subtraction" on their official revenue website ("https://tax.colorado.gov/retirees").

Reply:
I think this says it is the same as joint - https://tax.colorado.gov/deceased-taxpayer